src/labkey/Report.ts

```diff
@@ -222,7 +220,7 @@ export function getSessions(options: IGetSessionsOptions): XMLHttpRequest {
  * @private
  */
 function populateParams(options: IRequestExecuteOptions, isReport: boolean): IRequestExecuteParams {
-    let execParams: IRequestExecuteParams = {};
+    let execParams: any = {};
```

Comment:
@labkey-nicka I don't like having to drop this type, but didn't see another way to retain it here. Any thoughts?

Reply:
Did one better and just removed the `populateParams` method all together. It wasn't providing any value since it was used only within `requestExecute`.

I added a regression test case as well. Please take a look. Approving based on your approval. :-).

Reply:
Awesome! Thanks, Nick. That change looks good to me. I had dropped the last alpha package version into my other fb for platform core/package.json so that it could run against the medImmune tests on TeamCity and success! 
<img width="704" alt="Screen Shot 2020-04-23 at 9 16 42 PM" src="https://user-images.githubusercontent.com/6411206/80168156-c509ae00-85a7-11ea-84d2-5b9cefebbfea.png">

Reply:
Thanks for digging into this and applying the fix.